### PR TITLE
flake8 in dev docs

### PR DESCRIPTION
**Pull request by jgallowa07**
Dev Doc update

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -398,6 +398,25 @@ The unit test suite is in the ``tests`` directory. Tests are run using the
 
 from the project root to run the full test suite.
 
+It's useful to run the ``flake8`` CI tests *locally* before pushing a commit.
+To set this up use either ``pip`` or ``conda`` to install ``flake8``
+
+To run the test simply use::
+
+    $ flake --max-line-length 89 stdpopsim tests
```
Comment (jeromekelleher) [inline docs/development.rst]: should be flake8 here (the was a typo in my comment, sorry).

**Comment by jeromekelleher** [comment]
Great, thanks. Can you squash now please? For small things like this ``git commit --amend`` is really handy, saves the hassle of squashing.

**Comment by jgallowa07** [comment]
Learning so many git tricks @jeromekelleher! Is this good to merge now?

**Comment by andrewkern** [comment]
yeah the fact that you can `git commit --append` to an in progress PR is super helpful!

**Comment by jeromekelleher** [comment]
Excellent, thanks @jgallowa07! I worked pretty closely with a real git-guru a few years ago, ``git commit --amend`` is one of the things he showed me that really stuck. Super helpful, especially when dealing with annoying CI problems.

**Pull request merged**